packages/networks/src/networks/zkevm.ts

```diff
@@ -0,0 +1,79 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const zkevm: NetworkConfig = {
+  chain: 'zkevm',
+  description:
+    'Polygon zkEVM is a Layer 2 network of the Ethereum Virtual Machine (EVM), a zero-knowledge (ZK) rollup scaling solution.',
+  explorer: {
+    name: 'zkEVM (Polygonscan)',
+    urls: {
+      address: (address) => `https://zkevm.polygonscan.com/address/${address}`,
+      base: `https://zkevm.polygonscan.com/`,
+      token: (address, holder) =>
+        `https://zkevm.polygonscan.com/token/${address}?a=${holder}`,
+      transaction: (hash) => `https://zkevm.polygonscan.com/tx/${hash}`,
+    },
+  },
+  featured: false,
+  id: 1101,
+  isTestNetwork: false,
+  maxFreeClaimCost: 1,
+  multisig: '0xD62EF39c54d9100B17c8fA3C2D15e0262338AED0',
+  name: 'zkEVM (Polygon)',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'ETH',
+    symbol: 'ETH',
+    wrapped: '0x4F9A0e7FD2Bf6067db6994CF12E4495Df938E6e9',
+  },
+  previousDeploys: [],
+  // provider: 'https://rpc.unlock-protocol.com/1101',
+  provider: 'https://polygon-zkevm.drpc.org',
```

Comment:
```suggestion
  provider: 'https://rpc.unlock-protocol.com/1101',
```
@julien51 will need your help there too :)